core/tests/block.rs

```diff
@@ -34,13 +34,18 @@ use keychain::{BlindingFactor, ExtKeychain, Keychain};
 use std::sync::Arc;
 use util::{secp, RwLock, ToHex};
 
+// Setup test with AutomatedTesting chain_type;
+fn test_setup() {
+	global::set_local_chain_type(global::ChainTypes::AutomatedTesting);
+}
+
 fn verifier_cache() -> Arc<RwLock<dyn VerifierCache>> {
 	Arc::new(RwLock::new(LruVerifierCache::new()))
 }
 
 #[test]
 fn too_large_block() {
-	global::set_mining_mode(ChainTypes::AutomatedTesting);
+	global::set_local_chain_type(ChainTypes::AutomatedTesting);
```

Comment:
`test_setup()`or `global::set_local_chain_type(global::ChainTypes::AutomatedTesting);`?

Overall `test_setup()` don't seems necessary no?

Reply:
Good catch. I just want to use `test_setup()` to avoid all the repeated calls to `set_local_chain_type()` as this should be common across all the tests.